### ISSUE TITLE
win_copy failing at internal variable reference

### ISSUE THREAD

**Issue by Soolkiki** [ansible-collections/ansible.windows#111]
##### SUMMARY

I try to use the win_copy module and I get an exception thrown.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
win_copy

##### ANSIBLE VERSION

```
ansible 2.10.1
  config file = /builds/weigand-construction/wci-headquarters-infrastructure/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.6/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.6.9 (default, Jul 17 2020, 12:50:27) [GCC 8.4.0]

```

##### CONFIGURATION

```
DEFAULT_HOST_LIST(/builds/weigand-construction/wci-headquarters-infrastructure/ansible.cfg) = ['/builds/weigand-construction/wci-headquarters-infrastructure/environments/prod/hosts', '/builds/weigand-construction/wci-headquarters-infrastructure/environments/prod/headquarters_inventory.vmware.yml']
DEFAULT_ROLES_PATH(/builds/weigand-construction/wci-headquarters-infrastructure/ansible.cfg) = ['/builds/weigand-construction/wci-headquarters-infrastructure/roles']
HOST_KEY_CHECKING(/builds/weigand-construction/wci-headquarters-infrastructure/ansible.cfg) = False
INVENTORY_ENABLED(/builds/weigand-construction/wci-headquarters-infrastructure/ansible.cfg) = ['vmware_vm_inventory', 'script', 'ini']

```

##### OS / ENVIRONMENT

Windows Server 2019

##### STEPS TO REPRODUCE

I'm copying some powershell DSC modules I've written from ansible to my windows server with a playbook.

playbook

```
- hosts: VeeamBackupHosts
  gather_facts: no
  collections:
    - ansible.windows
  tasks:
    # Copy DSC Resources to Veeam Servers from Ansible
    - name: Copy DSC Resources
      ansible.windows.win_copy:
        src: files/powershell/veeam/dsc_resource_modules/
        dest: C:\Program Files\WindowsPowerShell\Modules\

```

##### EXPECTED RESULTS

Copy the files.

##### ACTUAL RESULTS

I execute the win_copy code, and it immediately throws an unhandled exception.

```paste below
The full traceback is:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/ansible/executor/task_executor.py", line 158, in run
    res = self._execute()
  File "/usr/local/lib/python3.6/dist-packages/ansible/executor/task_executor.py", line 663, in _execute
    result = self._handler.run(task_vars=variables)
  File "/root/.ansible/collections/ansible_collections/ansible/windows/plugins/action/win_copy.py", line 503, in run
    result.update(self._copy_single_file(file_src, dest, original_basename, file_dest,
UnboundLocalError: local variable 'original_basename' referenced before assignment
fatal: [wcveeamhq.wcdom.loc]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}
```

**Comment by jborean93** [comment]
This was fixed with https://github.com/ansible-collections/ansible.windows/pull/98, I'll do a release of this collection sometime today which will include the fix.

**Comment by Soolkiki** [comment]
Perfect! Thank you very much.

**Comment by jborean93** [comment]
`ansible.windows` has just been published at v1.0.1 with this fix.
